python-avd/pyavd/_eos_designs/structured_config/connected_endpoints/monitor_sessions.py

```diff
@@ -46,12 +46,13 @@ def monitor_sessions(self: AvdStructuredConfigConnectedEndpoints) -> list | None
                     "name": session["interface"],
                     "direction": get(session, "source_settings.direction"),
                 }
-                if (access_group := get(session, "source_settings.access_group")) is not None:
-                    source["access_group"] = {
-                        "type": access_group.get("type"),
-                        "name": access_group.get("name"),
-                        "priority": access_group.get("priority"),
-                    }
+                if get(merged_settings, "session_settings.access_group") is None:
```

Comment:
In eos_designs we should not just ignore stuff. Instead we should raise an error so the user is made aware.

Reply:
Added